samples/bluetooth/peripheral_mds/README.rst

```diff
@@ -33,54 +33,114 @@ The sample also includes the BAS functionalities.
 Metrics
 =======
 
-The sample shows how to capture user-specific metrics.
-It defines the following metrics:
+.. tabs::
 
-* ``button_3_press_count`` - The number of **Button 3** presses.
-* ``battery_soc_pct`` - The simulated battery level.
-* ``button_1_elapsed_time_ms`` - The time measured between two **Button 1** presses.
+   .. group-tab:: nRF52 and nRF53 DKs
 
-These metrics are defined in the :file:`samples/bluetooth/peripheral_mds/memfault_config/memfault_metrics_heartbeat_config.def` file.
-For more details about the metrics, see `Memfault: Collecting Device Metrics`_.
+      The sample shows how to capture user-specific metrics.
+      It defines the following metrics:
```

Comment:
Move this introductory text outside the tab, so you do not need to repeat it for both group-tabs. And do the same in other two sections below. Like this:

The sample shows how to capture...

.. tabs::